scripts/histmakers/mz_dilepton.py

```diff
@@ -461,6 +490,45 @@
     reverse_variations=args.reweightPixelMultiplicity
 )
 
+seed_data = 2 * args.randomSeedForToys
```

Comment:
can these two lines be moved inside the block "if args.nToysMC > 0:"

Reply:
Fixed.